<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoBang</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="4">
     <widget class="BoardUi" name="boardui" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>BLACK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLCDNumber" name="blackLCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>1800.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="whiteLCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>1800.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>WHITE</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>110</width>
        <height>219</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="regretButton">
        <property name="text">
         <string>Ask for Regret</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="giveupButton">
        <property name="text">
         <string>Give Up</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <widget class="QMenu" name="menuNew_Game">
     <property name="title">
      <string>&amp;New Game</string>
     </property>
     <addaction name="actionLocal_PvP"/>
     <addaction name="actionLocal_PvC"/>
     <addaction name="actionOnline_PvP"/>
    </widget>
    <addaction name="menuNew_Game"/>
    <addaction name="actionSave_Game"/>
    <addaction name="actionLoad_Game"/>
    <addaction name="actionQuit_Application"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLocal_PvP">
   <property name="text">
    <string>&amp;Local PvP</string>
   </property>
  </action>
  <action name="actionLocal_PvC">
   <property name="text">
    <string>Local &amp;PvC</string>
   </property>
  </action>
  <action name="actionOnline_PvP">
   <property name="text">
    <string>&amp;Online PvP</string>
   </property>
  </action>
  <action name="actionSave_Game">
   <property name="text">
    <string>&amp;Save Game</string>
   </property>
  </action>
  <action name="actionLoad_Game">
   <property name="text">
    <string>&amp;Load Game</string>
   </property>
  </action>
  <action name="actionQuit_Application">
   <property name="text">
    <string>&amp;Quit Application</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BoardUi</class>
   <extends>QWidget</extends>
   <header>boardui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
